sdks/go/test/load/sideinput/sideinput.go

```diff
@@ -51,11 +54,17 @@ func parseSyntheticConfig() synthetic.SourceConfig {
 	}
 }
 
+// impToKV just turns an impulse signal into a KV instead of
+// adding a single value input version of RuntimeMonitor
+func impToKV(imp []byte, emit func([]byte, []byte)) {
+	emit(imp, imp)
+}
+
 type doFn struct {
```

Comment:
I know this predates the change, but would you mind giving this a meaningful name? Maybe something like firstNKVPairs

Reply:
The focus for this one is that it uses an iterator, so calling that one out, so it's easier to add a Dictionary based one later.